src/fswAlgorithms/effectorInterfaces/thrMomentumDumping/thrMomentumDumping.c

```diff
@@ -69,34 +65,19 @@ void Reset_thrMomentumDumping(thrMomentumDumpingConfig *configData, uint64_t cal
         _bskLog(configData->bskLogger, BSK_ERROR, "Error: thrMomentumDumping.thrusterImpulseInMsg wasn't connected.");
     }
 
-    /*! - read in number of thrusters installed and maximum thrust values */
     localThrusterData = THRArrayConfigMsg_C_read(&configData->thrusterConfInMsg);
     configData->numThrusters = localThrusterData.numThrusters;
+
     for (i=0;i<configData->numThrusters;i++) {
         configData->thrMaxForce[i] = localThrusterData.thrusters[i].maxThrust;
     }
 
-    /*! - reset dumping counter */
     configData->thrDumpingCounter = 0;
-
-    /*! - zero out thruster on time array */
     mSetZero(configData->thrOnTimeRemaining, 1, MAX_EFF_CNT);
 
-    /*! - set the time tag of the last Delta_p message */
```

Comment:
restore original code.  This module has very specific reset behavior.  Let's not change this.